README.md

```diff
@@ -309,6 +309,10 @@ The following options are supported by all filters except `Callback` and `Finder
 - `mode` (`string`, defaults to `OR`) The conditional mode to use when searching for
   multiple values. Valid values are `OR` and `AND`.
 
+#### `Finder`
+
+- `finder` (`string`) The name of the finder method to use, without the 'find' prefix.
```

Comment:
You can't use `'finder' => 'findByMonth'`. Just link to docs, in any case if they know know how to write / use custom finder they will have to refer the cakephp docs, we are not going to duplicate them here.